statistics/bootstrap.go

```diff
@@ -188,14 +189,14 @@ func initStatsBuckets4Chunk(ctx sessionctx.Context, tables statsCache, iter *chu
 			var err error
 			lower, err = d.ConvertTo(ctx.GetSessionVars().StmtCtx, &column.Info.FieldType)
 			if err != nil {
-				log.Debugf("decode bucket lower bound failed: %s", errors.ErrorStack(err))
+				log.Debug("Decode bucket lower bound failed", zap.String("error stack", errors.ErrorStack(err)))
 				delete(table.Columns, histID)
 				continue
 			}
 			d = types.NewBytesDatum(row.GetBytes(6))
 			upper, err = d.ConvertTo(ctx.GetSessionVars().StmtCtx, &column.Info.FieldType)
 			if err != nil {
-				log.Debugf("decode bucket upper bound failed: %s", errors.ErrorStack(err))
+				log.Debug("Decode bucket upper bound failed", zap.String("error stack", errors.ErrorStack(err)))
```

Comment:
`zap.Error(err)` should have contained the error stack now, so we can directly use `zap.Error(err)` and remove those `errors.ErrorStack(err)` now? @lysu please confirm this.

Reply:
yes, zap.Error(err) will contain error stack if err is Traced by pingcap/errors

Reply:
Updated.
PTAL~